tests/providers/amazon/aws/operators/test_sagemaker_processing.py

```diff
@@ -0,0 +1,154 @@
+        self.assertEqual(self.sagemaker.config['ProcessingResources']['ClusterConfig']['VolumeSizeInGB'],
+                         int(self.sagemaker.config['ProcessingResources']['ClusterConfig']['VolumeSizeInGB']))
+        self.assertEqual(self.sagemaker.config['StoppingCondition']['MaxRuntimeInSeconds'],
+                         int(self.sagemaker.config['StoppingCondition']['MaxRuntimeInSeconds']))
+
+    @mock.patch.object(SageMakerHook, 'get_conn')
+    @mock.patch.object(SageMakerHook, 'create_processing_job')
+    def test_execute(self, mock_processing, mock_client):
+        mock_processing.return_value = {'ProcessingJobArn': 'testarn',
+                                        'ResponseMetadata': {'HTTPStatusCode': 200}}
+        self.sagemaker.execute(None)
+        mock_processing.assert_called_once_with(create_processing_params,
+                                                wait_for_completion=False,
+                                                check_interval=5,
+                                                max_ingestion_time=None
+                                                )
+
+    @mock.patch.object(SageMakerHook, 'get_conn')
+    @mock.patch.object(SageMakerHook, 'create_processing_job')
+    def test_execute_with_failure(self, mock_processing, mock_client):
+        mock_processing.return_value = {'ProcessingJobArn': 'testarn',
+                                        'ResponseMetadata': {'HTTPStatusCode': 404}}
+        self.assertRaises(AirflowException, self.sagemaker.execute, None)
+# pylint: enable=unused-argument
+
+    @mock.patch.object(SageMakerHook, "get_conn")
+    @mock.patch.object(SageMakerHook, "list_processing_jobs")
+    @mock.patch.object(SageMakerHook, "create_processing_job")
+    def test_execute_with_existing_job_increment(
+        self, mock_create_processing_job, mock_list_processing_jobs, mock_client
+    ):
+        self.sagemaker.action_if_job_exists = "increment"
+        mock_create_processing_job.return_value = {"ResponseMetadata": {"HTTPStatusCode": 200}}
+        mock_list_processing_jobs.return_value = [{"ProcessingJobName": job_name}]
+        self.sagemaker.execute(None)
+
+        expected_config = create_processing_params.copy()
+        # Expect to see ProcessingJobName suffixed with "-2" because we return one existing job
+        expected_config["ProcessingJobName"] = f"{job_name}-2"
+        mock_create_processing_job.assert_called_once_with(
+            expected_config,
+            wait_for_completion=False,
+            check_interval=5,
+            max_ingestion_time=None,
+        )
+
+    @mock.patch.object(SageMakerHook, "get_conn")
+    @mock.patch.object(SageMakerHook, "list_processing_jobs")
+    @mock.patch.object(SageMakerHook, "create_processing_job")
+    def test_execute_with_existing_job_fail(
+        self, mock_create_processing_job, mock_list_processing_jobs, mock_client
+    ):
+        self.sagemaker.action_if_job_exists = "fail"
```

Comment:
Maybe add another simple test to check if the validation for `action_if_job_exists` works as expected.

Reply:
Done.